prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+   <a href="file:///C:/Users/Mitchell/repos/prep-91621-code-solutions/prep-html-links/contact-us/contact-us.html"> <div> Click here to go to the contact page </div> </a>
```

Comment:
Here you can use a relative path to the `contact-us.html` file inside of the `contact-us` directory.

```html
<a href="contact-us/contact-us.html"> ... </a>
```